test/runtests.jl

```diff
@@ -424,6 +427,9 @@ cd(@__DIR__) do
         Test.record(o_ts, fake)
         Test.pop_testset()
     end
+
+    open(io -> write_testset_json(io, o_ts), "results.json", "w")
```

Comment:
Should we guard this by something like `Base.get_bool_env("CI", false)`

Reply:
I imagine this isn't much use to people running tests locally, I think we might as well. Pushed.